src/main/java/com/microsoft/jenkins/azuread/AzureCachePool.java

```diff
@@ -37,6 +37,7 @@ public static AzureCachePool get(GraphServiceClient<Request> azure) {
                     try {
                         DirectoryObjectCollectionWithReferencesPage collection = azure
                                 .users(oid)
+                                // TODO asGroup isn't working json error, and neither is $filter on securityEnabled
```

Comment:
see https://stackoverflow.com/questions/67982024/how-do-you-retrieve-transitive-groups-of-user-with-java-microsoft-graph-sdk